output/report.go

```diff
@@ -29,29 +29,23 @@ import (
 
 // Reporter is responsible for outputting test results
 type Reporter interface {
-	Stream(config *plan.Config, tests <-chan execute.TestResponse) Summary
-}
-
-// ReporterFunc streams test results to the console
-type ReporterFunc func(config *plan.Config, tests <-chan execute.TestResponse) Summary
-
-// Stream allows ReporterFunc to fulfill the Reporter interface
-func (f ReporterFunc) Stream(config *plan.Config, tests <-chan execute.TestResponse) Summary {
-	return f(config, tests)
+	Start(config *plan.Config) error
```

Comment:
config doesn't change over the course of the reporter - shouldn't we just instantiate the reporter with a config instead of passing it into each of these methods?